cassiopeia/solver/ILPSolver.py

```diff
@@ -90,6 +90,7 @@ def __init__(
         self.seed = seed
         self.mip_gap = mip_gap
 
+    @logger.namespaced("ILPSolver")
     def solve(
```

Comment:
Can we also log argument parameters here as well? Like the convergence time, LCA cutoff, cell cutoff, etc?

Reply:
Sure. I'll added in the function.